packages/strapi-plugin-content-type-builder/admin/src/translations/es.json

```diff
@@ -1,20 +1,76 @@
 {
   "attribute.boolean": "Booleano",
+  "attribute.boolean.description": "Sí o no, 1 o 0, verdadero o falso",
+  "attribute.component": "Componente",
+  "attribute.component.description": "Grupo de campos que puedes repetir o usar",
   "attribute.date": "Fecha",
-  "attribute.email": "Email",
+  "attribute.date.description": "Un calendario con horas, minutos y segundos, minutes and seconds",
```

Comment:
Forgot to remove "minutes and seconds"

Reply:
Diff's just look strange sometimes, it wasn't deleted just moved `" attribute.email " : " Coreo Electrónico " ,`

Reply:
Yes, I realized that afterwards haha. This comment is regarding the next line though :)